longhaul/longhaul.md

```diff
@@ -0,0 +1,235 @@
+# IoT SDK Long Haul Technical Spec
```

Comment:
Something we need to think about here is how we log.  Running for 2 weeks we need to make sure going through the logs has to be very clear in what's happening.